interfaces/devdoc/srw_lock_requirements.md

```diff
@@ -83,6 +99,23 @@ MOCKABLE_FUNCTION(, void, srw_lock_acquire_shared, SRW_LOCK_HANDLE, handle);
 
 **SRS_SRW_LOCK_02_026: [** If `do_statistics` is `true` and the timer created has recorded more than `TIME_BETWEEN_STATISTICS_LOG` seconds then statistics will be logged and the timer shall be started again. **]**
 
+### srw_lock_try_acquire_shared
+```c
+MOCKABLE_FUNCTION(, int, srw_lock_try_acquire_shared, SRW_LOCK_HANDLE, handle);
+```
+
+`srw_lock_try_acquire_shared` attempts to acquires the SRWLOCK in shared (read) mode.
```

Comment:
>acquires [](start = 42, length = 8)

acquire

Reply:
Fixed.

---
In reply to: [461957008](https://github.com/Azure/azure-c-pal/pull/20#discussion_r461957008) [](ancestors = 461957008)